### PR TITLE
Adding a bit of doc around the new debian packages.

### PR DESCRIPTION
**Pull request by maximelb**
## Description of the change

Adding some documentation around the new debian packages.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: docs/deploy_sensor.md [hunk 1]
```diff
@@ -77,6 +77,32 @@ installation key in an alternate place in the following order:
...
+
+The deb package will install the LimaCharlie sensor using a `systemd` service, or if unavailable a `system V` service.
+
+The installation key is required by the installer via the `debconf` configuration mechanism. By default, installing
+the package interactively will request the installation key via a local command/GUI interface. To perform large
+scale installations, we recommend setting the installation key programmatically.
+
+Installating interactively:
+```
+dpkg -i limacharlie.deb
```
Comment (amrikr) [inline docs/deploy_sensor.md:92]:
```suggestion
sudo dpkg -i limacharlie.deb
```

**Comment by amrikr** [comment]
I ran into problems trying to uninstall using `sudo dpkg -c limacharlie.deb`

It shows the following output:

```
administrator@administrator-virtual-machine:~/Downloads/lc-deb-inst$ sudo dpkg -c limacharlie.deb 
drwxr-xr-x root/root         0 2022-02-14 17:34 ./
drwxr-xr-x root/root         0 2022-02-14 17:34 ./bin/
-rw-r--r-- root/root    540392 2022-02-14 17:34 ./bin/rphcp
administrator@administrator-virtual-machine:~/Downloads/lc-deb-inst$ 
```

It does not appear to stop the running service nor remove the files from disk.

**Comment by maximelb** [comment]
> I ran into problems trying to uninstall using `sudo dpkg -c limacharlie.deb`
> 
> It shows the following output:
> 
> ```
> administrator@administrator-virtual-machine:~/Downloads/lc-deb-inst$ sudo dpkg -c limacharlie.deb 
> drwxr-xr-x root/root         0 2022-02-14 17:34 ./
> drwxr-xr-x root/root         0 2022-02-14 17:34 ./bin/
> -rw-r--r-- root/root    540392 2022-02-14 17:34 ./bin/rphcp
> administrator@administrator-virtual-machine:~/Downloads/lc-deb-inst$ 
> ```
> 
> It does not appear to stop the running service nor remove the files from disk.

It's not `-c`, it's `-i`.

File: docs/deploy_sensor.md [hunk 2]
```diff
@@ -77,6 +77,32 @@ installation key in an alternate place in the following order:
...
+
+The installation key is required by the installer via the `debconf` configuration mechanism. By default, installing
+the package interactively will request the installation key via a local command/GUI interface. To perform large
+scale installations, we recommend setting the installation key programmatically.
+
+Installating interactively:
+```
+sudo dpkg -i limacharlie.deb
+```
+
```
Comment (amrikr) [inline docs/deploy_sensor.md:94]:
```suggestion
Uninstalling interactively:

sudo dpkg -r limacharlie

```

File: docs/deploy_sensor.md [hunk 3]
```diff
@@ -77,6 +77,32 @@ installation key in an alternate place in the following order:
...
+Installing and setting the installation key programmatically:
+```
+echo "limacharlie limacharlie/installation_key string INSTALLATION_KEY_HERE" | debconf-set-selections && dpkg -i limacharlie.deb
```
Comment (amrikr) [inline docs/deploy_sensor.md:97]:
```suggestion
echo "limacharlie limacharlie/installation_key string INSTALLATION_KEY_HERE" | sudo debconf-set-selections && sudo dpkg -i limacharlie.deb
```

**Review by amrikr — APPROVED** [review]
👍🏽

**Pull request merged**